server/api/src/rsDataObjRename.cpp

```diff
@@ -47,7 +47,7 @@
 #include <cstring>
 #include <chrono>
 
-using logger = irods::experimental::log;
+using log_api = log_api;
```

Comment:
I see now. Some other header is defining this alias, right?

Let's explicitly define the alias so there's no dependency on a header providing it.
```cpp
using log_api = irods::experimental::log::api;
```